config/initializers/diaspora_federation.rb

```diff
@@ -0,0 +1,8 @@
+# configure the federation engine
+DiasporaFederation.configure do |config|
+  # the pod url
+  config.server_uri = AppConfig.pod_uri
+
+  # the class to be used for a person
+  config.person_class = Person.to_s
```

Comment:
Yes, I'm just saying it's safer to do in the accessor, since you then 1) don't need that knowledge 2) aren't confused by it since it's common not to in the Rails API (e.g. `has_many :foo, class: Bar`)